### PR TITLE
[Issue-69]: Friend system

### PR DESCRIPTION
**Pull request by ltuffery**
Systeme d'amies

### CODE REVIEW

File: srcs/django_code/transcend/users/views.py [hunk 1]
```diff
@@ -25,11 +25,30 @@ class UserView(APIView):
...
+        serializer = UserSerializer(request.user, many=False)
         return Response(serializer.data, status=status.HTTP_200_OK)
 
     def put(self, request):
         user = User.objects.get(email=request.user.email)
         user.avatar = request.data['avatar']
         user.save()
         return Response({'message': 'Image updated'}, status=status.HTTP_200_OK)
+
+import sys
```
Comment (Rreyth) [inline srcs/django_code/transcend/users/views.py:37]: DELETE

**Review by Rreyth — CHANGES_REQUESTED** [review]
views.py 37 -> delete useless import

File: srcs/django_code/transcend/users/views.py (hunk 1 above)
Comment (ltuffery) [reply srcs/django_code/transcend/users/views.py]: [674f199](https://github.com/Rreyth/get-transcended/pull/70/commits/674f199caaac5f276d960635d4b9c9905adb50b8) fix ici

**Review by Rreyth — APPROVED** [review]
knuckles

**Pull request merged**